packit_service/worker/mixin.py

```diff
@@ -289,10 +289,17 @@ def branches(self) -> List[str]:
         You can retrigger the update by adding a comment (`/packit propose-downstream`) into this issue.
         """
         branches = []
+        branch_regex = re.compile(r"\s*\| `(\S+)` \|")
         issue = self.data.project.get_issue(self.data.issue_id)
         for line in issue.description.splitlines():
-            if m := re.match(r"\s*\| `(\S+)` \|", line):
-                branches.append(m[1])
+            if m := branch_regex.match(line):
+                if m[1] not in branches:
+                    branches.append(m[1])
+        for comment in issue.get_comments():
+            for line in comment.body.splitlines():
+                if m := branch_regex.match(line):
+                    if m[1] not in branches:
```

Comment:
if `branches` was a set you wouldn't need to check

Reply:
I wasn't sure if the order wasn't important, but I suppose it doesn't matter, I'll use a set.